ecosystem/sep-0038.md

```diff
@@ -30,7 +30,9 @@ An anchor must define the location of their `ANCHOR_QUOTE_SERVER` in their [`ste
 
 ### Authentication
 
-All endpoints require authentication in the form of a [SEP-10](sep-0010.md) JSON Web Token (JWT) in the `Authorization` header of the request. 
+This protocol's endpoints use authentication in the form of a [SEP-10](sep-0010.md) JSON Web Token (JWT) in the `Authorization` header of the request.
+
+While it is mandatory for [`POST /quote`](#post-quote) and [`GET /quote/:id`](#get-quote), it is optional for the remaining endpoints. When optional, the JWT token can be used to personalize the respective responses.
```

Comment:
I think the document would benefit from explicitly stating the authentication in the section of each endpoint. We do this in SEP-30 and it makes it ultra clear in the section for each endpoint the types of authentication that's available or required. See this for an example:

https://github.com/stellar/stellar-protocol/blob/799105e14c4cf2b963870760ffade1132928da50/ecosystem/sep-0030.md#L498-L499

Reply:
Done in https://github.com/stellar/stellar-protocol/pull/1144/commits/d943c889c06a25c51df79bdc48a665187ca86912.